src/Models/Collection.php

```
@@ -0,0 +1,131 @@
+     * @var class-string<T>
+     */
+    protected static string $itemType;
+
+    protected static ?CollectionSchemaInterface $schema = null;
+
+    public function count(): int
+    {
+        return count($this->models);
+    }
+
+    public function jsonSerialize(): array
```

PHPStan (2.1.15)

Method OpenFGA\Models\Collection::jsonSerialize() return type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)